test/protocol_test.dart

```diff
@@ -31,15 +31,6 @@ void main() {
           process, "Protocol message contained an invalid tag (zero).");
       await process.kill();
     });
-
-    test("without shutting down the compiler", () async {
```

Comment:
We should add a test that asserts that the compiler *does* shut down in this case.